olp-cpp-sdk-core/include/olp/core/cache/CacheSettings.h

```diff
@@ -19,9 +19,8 @@
 
 #pragma once
 
-#include <string>
-
 #include <boost/optional.hpp>
+#include <string>
```

Comment:
Not entirely correct.
System headers are separated from other libraries. Please roll back.